altapay.php

```diff
@@ -1414,7 +1422,7 @@ public function renderTerminalList()
         $helper->currentIndex = AdminController::$currentIndex . '&configure=' . $this->name;
         $helper->orderBy = 'id_terminal';
         $helper->orderWay = 'ASC';
-        $content = Altapay_Models_Terminal::getTerminals();
+        $content = Altapay_Models_Terminal::getTerminals((int) $this->context->shop->id);
```

Comment:
You are validating the the shop_id in the function, why is it being casted now?

Reply:
Done